cmd_args.py

```diff
@@ -126,15 +126,16 @@ def parse_args(root_dir):
 # =============================================================================
     # Dataset args
     parser.add('--pad', required=True, type=type_or_none_fn(int),
-               help="Padding around intervals")
+               help="Number of additional bases to add as padding \
```

Comment:
should it match the --pad from calculate_baseline_metrics ? Then, should we explicitly mention that in scripts that use --pad ?

Reply:
Updated the --pad description in all scripts that use it.